data/tutorials/guides/1wf_04_multicore_ready.md

```diff
@@ -0,0 +1,303 @@
+---
+id: multicore-transition
+title: Transitioning to Multicore with TSan
+description: >
+  Learn to make your OCaml code multicore ready with ThreadSanitizer
+category: "Guides"
+---
+
+
+# Transitioning to Multicore with ThreadSanitizer
+
+The 5.0 release brought multicore `Domain`-based parallelism to the
+OCaml language, and with it new opportunities for data races. In this
+guide, we will study a step-wise workflow that utilizes the
+ThreadSanitizer (TSan) tool to help make your OCaml code 5.x ready.
```

Comment:
I'm not sure the intro given here is specific enough. Maybe there's the possibility to elaborate what ThreadSanitizer is and why one would use it? It's probably not clear for a first-time reader what "5.x ready" means  and what ThreadSanitizer is at this point. There's just this little hint about data races.

Reply:
Point well taken. I'll elaborate some more :+1:

Reply:
There's now a revised attempt in b3f8f25